Library/Formula/gstreamer.rb

```diff
@@ -28,6 +28,7 @@ class Gstreamer < Formula
   # Snow Leopard)
   # https://bugzilla.gnome.org/show_bug.cgi?id=756136
   patch :DATA if MacOS.version <= :snow_leopard
+  patch :DATA if MacOS.version <= :mountain_lion
```

Comment:
i.e. the latter replaces the former, so it might as well actually replace the former.